src/legacy/ui/public/vis/components/tooltip/_pointseries_tooltip.html

```diff
@@ -1,10 +1,11 @@
 <table>
   <tbody>
     <tr ng-repeat="detail in details" >
-      <td class="visTooltip__label">{{detail.label}}</td>
+      <td class="visTooltip__label">
+        <div class="visTooltip__labelContainer">{{detail.label}}</div>
```

Comment:
I would just suggest just using the EUI utility CSS class for this `.eui-textOverflowWrap`. It already has the nice fallbacks and it will stay up to date in case there needs to be other browser fallbacks in the future.

Reply:
Hey @cchaos I've seen that `euiTextOverflowWrap` is written as:
```
@mixin euiTextOverflowWrap  {
  @include internetExplorerOnly {
    word-break: break-all;
  }
  overflow-wrap: break-word;
}
```
where for IE you specify `break-all` that breaks every word also the shorter ones causing this (see how it breaks also the `Descending` word):

**with euiTextOverflowWrap**
<img width="486" alt="Screenshot 2019-07-23 at 11 24 54" src="https://user-images.githubusercontent.com/1421091/61700566-26f96680-ad3d-11e9-9d92-164613a4c912.png">

**with  word-wrap: break-word**
<img width="457" alt="Screenshot 2019-07-23 at 11 25 44" src="https://user-images.githubusercontent.com/1421091/61700704-3ed0ea80-ad3d-11e9-9e53-4edcc29cef0a.png">

Reply:
Ugh! `word-wrap` `word-break` `overflow-wrap`..... They need to stop renaming/adding new names to these properties. 

You are correct. It looks like we have been using `word-break: break-word` which IE doesn't support so our fallback had been `word-break: break-all`:

https://github.com/elastic/eui/blob/a9f11ed66b3bde0405fb9e3680e3a3436c55949f/src/global_styling/utility/_utility.scss#L27-L30

Which I probably just thought that's what the fallback should be. Perhaps we need to re-evaluate what properties we're using and the appropriate fallbacks.

Would you mind just going ahead and sticking with the EUI utility class, and we'll be sure to fix those fallbacks?

Reply:
done in 33bb3d7
@cchaos Do you want me to open an issue on EUI to fix that?

Reply:
I've already fixed this in https://github.com/elastic/eui/pull/2157 and it should be in EUI v.13.1.0. It removes the `.eui-textOverflowWrap` class in favor of the `.eui-textBreakWord` class which was updated to work across all browsers.